config/initializers/mock_ldap_config_check.rb

```diff
@@ -0,0 +1,19 @@
+should_use_mock_ldap = ::EnvironmentVariable.is_true('USE_TEST_PALS_LDAP')
+
+env_is_production = Rails.env.production?
+
+district_is_demo = (ENV.fetch('DISTRICT_KEY') == 'demo')
```

Comment:
Let's use`PerDistrict` for this, either put a method there or something like `PerDistrict.new.district_key == PerDistrict.DEMO`